infra/prod/_modules/storage_account/storage_account_fims.tf

```diff
@@ -0,0 +1,16 @@
+module "storage_account_fims" {
+  source = "github.com/pagopa/terraform-azurerm-v3//storage_account?ref=v7.76.0"
+
+  name                          = "${var.prefix}${var.env_short}${var.domain}st"
+  domain                        = upper(var.domain)
```

Comment:
what is the purpose of this `domain` variable?

Reply:
Actually we do not need it so I removed it [4dfde46](https://github.com/pagopa/io-fims/pull/24/commits/4dfde46e4f1109f98bcfc163cf88e0579a24be68)